core/src/main/java/discord4j/core/object/entity/channel/ThreadChannel.java

```diff
@@ -104,14 +117,14 @@ public Mono<ThreadMember> getMember(Snowflake userId) {
     /**
      * Requests to retrieve the member of this thread, using the given retrieval strategy.
      *
-     * @param userId The ID of the user.
+     * @param userId            The ID of the user.
```

Comment:
What happen with the spaces? xd

Reply:
Wut xD ? I don't know what happened xD

Reply:
I probably did reformat the whole file, this is an habit of mine and I have issues dropping it.
I will revert those changes to restore the old indentation. I'm not at home I'll do it later.